<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Clientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Regitro de Clientes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombres:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Materno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dirección:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Teléfono:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuitar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblClientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>771</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dirección</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Teléfono</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
